tox.ini

```diff
@@ -1,60 +1,37 @@
 [tox]
 mintoxversion = 2.3
-envlist = check_readme,flake8,covclean,py38,coverage
+envlist = flake8,py38
 
 [testenv]
 passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH
 deps =
-	pip-tools
-	coverage
+    pip-tools
+    coverage
 commands =
-	pip-sync requirements/base.txt requirements/tests.txt
-	pip install -e .
-	coverage run -m py.test {posargs:tests}
-
-[testenv:covclean]
-basepython = python3.8
-skip_install = true
-deps = coverage
-commands = coverage erase
-
-[testenv:coverage]
-basepython = python3.8
-skip_install = true
-deps = coverage
-commands =
-	coverage combine
-	coverage report
-
+    pip-sync requirements/tests.txt
+    pip install -e .
+    coverage run -m py.test {posargs:tests}
 
 [testenv:flake8]
 basepython = python3.8
 skip_install = true
 deps =
-	-rrequirements/tests.txt
+    -rrequirements/tests.txt
 commands =
-	flake8
-	#isort --check-only --recursive --quiet --project d3a --line-width 99 src tests setup.py fabfile.py
+    flake8
+    #isort --check-only --recursive --quiet --project d3a --line-width 99 src tests setup.py fabfile.py
 
 [flake8]
 max-line-length = 99
 
-[testenv:check_readme]
-skip_install = true
-deps = readme_renderer
-commands = python setup.py check --restructuredtext --strict
-
 [testenv:ci]
 basepython = python3.8
-passenv = LANG TERM LANGUAGE LC_ALL LD_LIBRARY_PATH
-deps =
-	pip-tools
-	coverage
```

Comment:
These values are already set in the base `testenv` (which is inherited by all other environments) so they can be safely removed.